src/main/java/frc/team4909/robot/LidarLitePWM.java

```diff
@@ -0,0 +1,66 @@
+package frc.team4909.robot;
+
+import edu.wpi.first.wpilibj.Counter;
+// import edu.wpi.first.wpilibj.DigitalSource;
+
+public class LidarLitePWM {
+	/*
+	 * Adjust the Calibration Offset to compensate for differences in each unit.
+	 * We've found this is a reasonably constant value for readings in the 25 cm to
+	 * 600 cm range. You can also use the offset to zero out the distance between
+	 * the sensor and edge of the robot.
+	 */
```

Comment:
Nice comment! Well describes the meaning and that the value was determined empirically.